### PR TITLE
Display interview question details and implement toggle button

### PR DESCRIPTION
**Pull request by kyi193**
Hey guys, I implemented the details section of the interview page and styling. You can click the answer or question button to toggle between the question and solution.

Screen Shot 2020-11-16 at 8 24 06 PM
Screen Shot 2020-11-16 at 8 24 10 PM

### CODE REVIEW

File: client/src/themes/theme.js [hunk 1]
```diff
@@ -200,6 +201,31 @@ export const useStyles = makeStyles((theme) => ({
   textMarginLeft: {
     marginLeft: '10px'
   },
+  questionAnswerButtonContainer: {
```
Comment (dahrk) [inline client/src/themes/theme.js:204]: It feels like you can merge answerContainer and QAButtonContainer or do some kind of inherit?

File: client/src/components/Buttons.js [hunk 2]
```diff
@@ -36,3 +36,30 @@ export const StartDashboardButton = withStyles({
     marginTop: '30px'
   },
 })(Button);
+
+
+export const InterviewDetailButtonHighlighted = withStyles({
```
Comment (dahrk) [inline client/src/components/Buttons.js:41]: Is there anyway we can re-use buttons and just have different labels on them? The reason for cohesive styling is so we can re-use as much code as possible - i.e. one button with it's own state that chooses color based on whether it's toggled on or off

File: client/src/Components/InterviewQuestionDetails.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
+import React, { useContext, useState } from "react";
```
Comment (dahrk) [inline client/src/Components/InterviewQuestionDetails.js:1]: We might not need useContext here, it's good to try and minimize imports as a best practice/bring them in as you need them.

**Review by dahrk — COMMENTED** [review]
Left some comments for you, let me know what you think. Hoping to push your skills as a front-end dev :)

**Comment by kyi193** [comment]
Okay, I went ahead and fixed all issues above. No more crazy conditional rendering on the details component!

**Pull request merged**